app/models/report.rb

```diff
@@ -0,0 +1,8 @@
+class Report < ApplicationRecord
+  belongs_to :reporter, class_name: 'User'
+  belongs_to :project
+
+  validates :reason, presence: true
+
+  enum status: [ :active, :resolved ]
```

Comment:
A minor thing. These two states look sufficient for now and I'm not sure if we'll need more at the moment. But just to make it flexible, can we use a hash instead of an array?